libbeat/cmd/instance/metrics.go

```diff
@@ -30,3 +27,12 @@ func (memstatsVar) Visit(m monitoring.Mode, V monitoring.Visitor) {
 		monitoring.ReportInt(V, "gc_next", int64(stats.NextGC))
 	}
 }
+
+func reportInfo(_ monitoring.Mode, V monitoring.Visitor) {
+	V.OnRegistryStart()
+	defer V.OnRegistryFinished()
+
+	delta := time.Since(startTime)
+	uptime := int64(delta.Seconds())*1000 + int64(delta.Nanoseconds()/1000000)
```

Comment:
A `time.Duration` is an `int64` giving the number of nanoseconds so you should just need to divide by `time.Millisecond` to convert to milliseconds.

`uptimeMillis := int64(time.Since(startTime)/time.Millisecond)`